src/content/lodash/array/fromPairs/vanilla.js

```diff
@@ -1,8 +1,2 @@
-const fromPairs = pairs =>
-  pairs.reduce((cache, pair) => {
-    cache[pair[0]] = pair[1]
-    return cache
-  }, {})
-
-module.exports = fromPairs([['a', 1], ['b', 2]])
+module.exports = Object.fromEntries([['a', 1], ['b', 2]])
```

Comment:
Should we keep previous implementation, or add a comment about `Object.fromEntries` being pretty cutting-edge (e.g. it works only starting from node v12), or MDN link would be enough?

Reply:
Interesting questions, what do you think?

Reply:
We can keep both implementations with a clear comment that one way is preferred (due to performance and readability), but has some usage limitations due to being new, and the other one will work almost anywhere

Reply:
Sounds good to me. Also thank you again for picking up all of these. I was already thinking about adding a contributors section in the readme but with your contributions it has become necessary :)